### PR TITLE
Allow gpg watch user configuration dirs and user temp dirs

### PR DESCRIPTION
**Pull request by zpytela**

### CODE REVIEW

File: policy/modules/contrib/gpg.te [hunk 1]
```diff
@@ -166,6 +166,7 @@ userdom_manage_user_home_content_files(gpg_t)
 userdom_manage_user_home_content_dirs(gpg_t)
 userdom_filetrans_home_content(gpg_t)
 userdom_stream_connect(gpg_t)
+userdom_watch_tmp_dirs(gpg_t)
```
Comment (WOnder93) [inline policy/modules/contrib/gpg.te:169]: I wonder if this one won't be covered by `userdom_manage_all_user_tmp_content(gpg_t)` after #574 is merged.

Comment (WOnder93) [reply policy/modules/contrib/gpg.te:169]: And indeed it is:
```
$ macro-expander 'userdom_manage_all_user_tmp_content(gpg_t)' | grep watch
allow gpg_t user_tmp_type:dir { create open getattr setattr read write link unlink rename search add_name remove_name reparent rmdir lock ioctl watch watch_reads };
allow gpg_t user_tmp_type:file { create open getattr setattr read write append rename link unlink ioctl lock watch watch_reads };
allow gpg_t user_tmp_type:lnk_file { create getattr setattr read write append rename link unlink ioctl lock watch watch_reads };
```

**Comment by zpytela** [comment]
Resolved with merging #574.

**Comment by WOnder93** [comment]
And what about the `allow gpg_t gpg_secret_t:dir watch_dir_perms;` part? It doesn't seem to be covered by the other interfaces (there is just manage for files).

**Comment by zpytela** [comment]
I only checked those:
```
# sesearch -A -s gpg_t -p watch -t gpg_secret_t
allow gpg_t gpg_secret_t:file { append create getattr ioctl link lock open read rename setattr unlink watch watch_reads write };
allow gpg_t gpg_secret_t:lnk_file { append create getattr ioctl link lock read rename setattr unlink watch watch_reads write };
```
No dir present there.

**Comment by WOnder93** [comment]
Well, that's the point :) The original PR also added `allow gpg_t gpg_secret_t:dir watch_dir_perms`, which isn't obsoleted by #574.

**Pull request closed without merging**